packages/office-ui-fabric-react/src/components/Callout/CalloutPage.tsx

```diff
@@ -60,9 +60,6 @@ export class CalloutPage extends React.Component<IComponentDemoPageProps, any> {
             <p>Real-world examples of this implementation can be seen in administrative interfaces where a particularly difficult-to-understand concept is paired with the iconClassNames.info "i" icon. In this example, Callout - with its tip text - is opened when the user clicks on or hovers over the icon.</p>
           </div>
         }
-        related={
```

Comment:
Did this for sure remove the "also available in" links? Those still seem to show up locally for me when I tested against this branch.